cmd/daemon.go

```diff
@@ -110,7 +112,7 @@ func gcloudAdcSetup(ctx context.Context) error {
 		// already present skip
 		return nil
 	}
-	url := getenv("CEDANA_SERVER_URL", "https://sandbox.cedana.ai") + "/k8s/gcloud/serviceaccount"
+	url := getenv("CEDANA_URL", "https://sandbox.cedana.ai") + "/k8s/gcloud/serviceaccount"
```

Comment:
Should use viper to fetch this value instead of directly from env. It might exist in the config.